services/core-web/common/components/PartyAppointmentTable.js

```diff
@@ -85,14 +87,17 @@ const PartyAppointmentTable = (props) => {
       render: (status, record) => {
         if (isCore) {
           return (
-            <Field
-              id={`${record.rowName}.status`}
-              name={`${record.rowName}.status`}
-              component={renderConfig.SELECT}
-              data={statusColumns}
+            <Select
```

Comment:
Curious as to why we're switching from `<Field>` to `<Select>`?
Seeing as we're using `redux-form` and `<Field component={renderConfig.SELECT}>` throughout the rest of the application, I would suggest that we stick to that if we can

Reply:
Actually, that's something I wanted to discuss. I'll try harder to find a work around in the `Field`. Thanks.

Reply:
@simensma-fresh 
Could solve using the same `Field` component.